app/assets/stylesheets/finder_frontend.scss

```diff
@@ -233,16 +233,91 @@
     .result-info {
       @include core-16;
       vertical-align: baseline;
-      border-bottom: solid 1px $border-colour;
       margin: 0 0 $gutter-half 0;
-      padding-bottom: $gutter-half;
 
       .result-count {
         @include bold-48;
         margin-right: $gutter-one-third / 2;
       }
     }
 
+    .filters {
+      margin-top: $gutter-one-third;
+      margin-bottom: $gutter;
+    }
+
+    .facet-tag-group {
+      padding: 5px;
+
+      &:nth-child(odd) {
+        background-color: $grey-4;
+      }
+
+      .facet-tag-wrapper {
+        vertical-align: middle;
+        margin: 5px 0;
+
+        @include media(tablet) {
+          display: inline-block;
+        }
+      }
+
+      .facet-tag-wrapper:first-of-type .facet-tag-preposition:first-of-type {
+        font-weight: bold;
+
+        &::first-letter {
+          text-transform: uppercase;
+        }
+      }
+
+      .facet-tag-preposition {
+        padding: 0 5px 0 2px;
+        vertical-align: middle;
+        text-transform: lowercase;
+
+        @include media(tablet) {
+          display: table-cell;
+        }
+      }
+
+      .facet-tag {
+        display: block;
+        position: relative;
+        padding: 5px;
+        border: 1px solid $grey-1;
+        border-radius: 5px;
+        background-color: $grey-4;
+
+        .js-enabled & {
+          padding: 5px 5px 5px $gutter-two-thirds;
+        }
+
+        @include media(tablet) {
+          display: table-cell;
+        }
+
+        .facet-tag-text {
+          display: inline-block;
+          margin-left: 5px;
+        }
+
+        .remove-filter {
```

Comment:
The remove button makes for quite a small click target (on desktop and mobile). Did you consider making the entire facet tag clickable to remove?

Reply:
I had already added some extra padding around the button to increase the clickable area from the default size. I think making the whole tag clickable to remove is probably a separate conversation and shouldn't be blocking this PR.

Reply:
Let's make a note for when we look into turning this into a component.